docs/specs/multi-registry/multi_registry.md

```diff
@@ -0,0 +1,501 @@
+  independently, without requiring coordination with the OTEL project.
+- **Core Policy Enforcement**: The OTEL project should establish and enforce core policies (e.g., backward
+  compatibility) that all published registries must adhere to, ensuring consistency and reliability.
+- **Registry Accessibility**: Registries should be either private or public, depending on the owner’s needs, 
+  and accessible via a URL.
+- **Cross-Registry References**: References between different registries should be supported to facilitate
+  interoperability and integration across various registries.
+- **Clear Rules to Avoid Conflicts**: A well-defined set of rules should be established and enforced to prevent
+  conflicts among imported attributes and signals. Users are responsible for resolving any conflicts that cannot
+  be automatically resolved by the tooling.
+- **Circular Reference Handling**: Circular references between registries should be automatically detected,
+  reported, and rejected to maintain system integrity and prevent conflicts.
+- **Open Publication Format**: A published registry must adhere to a open format and packaging designed to ensure
+  stability, ease of exchange, and consumption by other registries or tools.
+- **Self-Contained**: A published registry should be self-contained to make the overall system more robust and
+  resilient to changes in dependencies.
+- **Version Discoverability**: Available versions of a published registry should be easily discoverable.
+- **Transparency for Telemetry Consumers**: Downstream consumers of telemetry data (e.g., backends and dashboards)
+  should never be exposed to conflicts within or between registries. They should see attributes and signals as
+  defined, without scope or conflict resolution directives, ensuring a consistent and reliable data experience.
+- **Community Support Tools**: The OTEL project should provide and maintain an extensible tool, Weaver, to
+  assist the community in creating, validating, resolving, packaging, and publishing registries. Weaver should
+  also support documentation and code generation for any published registry.
+- **Clear, Actionable, and Educational Error Messages**: Weaver should detect and report any errors or inconsistencies
+  in a registry, providing users with clear and actionable feedback.
+
+## Semantic Convention Registry Changes
+
+The following sections describe the changes needed in the semantic convention registry to support multiple registries.
+
+### Directory Structure
+
+The directory structure of a registry remains mostly unchanged, with the addition of a new file called
+`weaver_registry.yaml`. This file contains the registry metadata, such as the name, version, and description.
+It is required for published registries or registries with dependencies on other registries.
+
+```plaintext
+registry_root/
+  domain_1/
+  domain_2/
+  ...
+  resources/
+  weaver_registry.yaml
+```
+
+### `weaver_registry.yaml` File
```

Comment:
I think we should decide if/how it aligns with the 3-file-schema-approach we discussed in semconv tooling call:

* 3 files  
  * Existing file \- marker  
    * contains pointer to diff  
    * contains pointer to def  
  * Model file  
     * Contains the full set of defined semantic conventions for that version  
  * Migration file  
     * Contains transformations you can use to keep version compatible \*when they are non breaking differences\*  
      * Should be cohesive/stand-alone for transformation use case.

There seem to be a lot of intersection with what'd we put into the 2.0 of `schema.yaml` and we should figure out if we need another one for registry manifest.